src/configure/configurers/localGithubWorkflowConfigurer.ts

```diff
@@ -273,12 +272,9 @@ export class LocalGitHubWorkflowConfigurer implements Configurer {
 
                 let authorName = await LocalGitRepoHelper.GetHelperInstance(inputs.sourceRepository.localPath).getUsername();
                 let deployerName = 'GITHUBACTION';
-                let updateDeploymentLogPromise = (azureResourceClient as AppServiceClient).publishDeploymentToAppService(inputs.targetResource.resource.id, deploymentMessage, authorName, deployerName);
+                await (azureResourceClient as AppServiceClient).publishDeploymentToAppService(inputs.targetResource.resource.id, deploymentMessage, authorName, deployerName);
```

Comment:
Can we do sourcecontrol and deployment update in parallel?

Reply:
All the post steps are done in background. So I thought to serialize it to keep things simple. I have updated it to make the request in parallel